packages/amplication-data-service-generator/src/server/docker-compose/docker-compose.db.template.yml

```diff
@@ -1,13 +1 @@
 version: "3"
```

Comment:
link to the question I asked on the plugins repo PR
https://github.com/amplication/plugins/pull/2#issuecomment-1250137953